includes/Activation_Notice.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * Class Activation_Notice.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+use Google\Web_Stories\Traits\Assets;
+
+/**
+ * Class Activation_Notice.
+ */
+class Activation_Notice {
+	/**
+	 * Script handle.
+	 *
+	 * @var string
+	 */
+	const SCRIPT_HANDLE = 'web-stories-activation-notice';
+
+	use Assets;
+
+	/**
+	 * Activation flag instance.
+	 *
+	 * @var Activation_Flag
+	 */
+	protected $activation_flag;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Activation_Flag $activation_flag Activation flag instance.
+	 */
+	public function __construct(
+		Activation_Flag $activation_flag
+	) {
+		$this->activation_flag = $activation_flag;
+	}
+
+	/**
+	 * Initializes the plugin activation notice.
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
+		add_action( 'admin_notices', [ $this, 'render_notice' ] );
+		add_action( 'network_admin_notices', [ $this, 'render_notice' ] );
+	}
+
+	/**
+	 * Enqueues assets for the plugin activation notice.
+	 *
+	 * @param string $hook_suffix The current admin page.
+	 * @return void
+	 */
+	public function enqueue_assets( $hook_suffix ) {
+		if ( 'plugins.php' !== $hook_suffix || ! $this->activation_flag->get_activation_flag( is_network_admin() ) ) {
+			return;
+		}
+
+		/**
+		 * Prevent the default WordPress "Plugin Activated" notice from rendering.
+		 *
+		 * @link https://github.com/WordPress/WordPress/blob/e1996633228749cdc2d92bc04cc535d45367bfa4/wp-admin/plugins.php#L569-L570
+		 */
+		unset( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification, WordPress.VIP.SuperGlobalInputUsage
+
+		wp_enqueue_style(
```

Comment:
We register this font in 3 places now. 
https://github.com/google/web-stories-wp/blob/master/includes/Story_Post_Type.php#L290-L295
https://github.com/google/web-stories-wp/blob/master/includes/Dashboard.php#L171-L176

Can we register it globally or have a trait with a reusable method?
I dont like that they make different requests on the font css. Can we load all of roboto and google sans in once place and the request will be changed on other views?

Reply:
Great point. I think we really need to take a hard look at this.

We need different Roboto font weights and styles for each case, so it would probably be better to name them differently & keep them separate, instead of registering it globally.

This way, we make sure we only load the CSS we absolutely need for each screen.

Reply:
I disagree here. Loading another uncached styled stylesheet here is wasteful. 
I have broken this out to a new PR - https://github.com/google/web-stories-wp/pull/2963
Can you take a look here @swissspidy